src/Agent.Listener/Configuration/ServiceControlManager.cs

```diff
@@ -60,6 +60,11 @@ protected void CalculateServiceName(AgentSettings settings, string serviceNamePa
         protected void SaveServiceSettings()
         {
             string serviceConfigPath = IOUtil.GetServiceConfigFilePath();
+            if(File.Exists(serviceConfigPath))
+            {
+                IOUtil.DeleteFile(serviceConfigPath);
```

Comment:
Docs say if it exists, it's overwritten.  Why delete?
https://msdn.microsoft.com/en-us/library/ms143376(v=vs.110).aspx